tests/test_reader_binary.py

```diff
@@ -96,8 +96,6 @@ def _prepend_ivm(params):
 )
 
 _BAD_VALUES = (
-    # Only up to microsecond precision is supported (6 digits).
-    (b'\x69\xC0\x81\x81\x81\x80\x80\x80\xC7\x01', 'OVERFLOWING TIMESTAMP PRECISION'),
```

Comment:
Instead of just removing this, let's move it to `_TOP_LEVEL_VALUES` and verify that it is written with full precision.

Reply:
Done.